### DISCUSSION TITLE
How to use Limited privileges Auth with Realtime Database?

### DISCUSSION THREAD

**Discussion by tomykho** [kreait/firebase-php#624] [Q&A]
Is it possible to use `databaseAuthVariableOverride` with 5.x in this SDK like in 4.x `asUser` function?

https://firebase.google.com/docs/database/admin/start#authenticate-with-limited-privileges

**Comment by tomykho** [comment]
Should I use `auth_variable_override` in database URI?

**Comment by jeromegamez** [comment]
Both are currently not possible with the 5.x version of the SDK. I'll see if I can port the `asUser` functionality back 🤞
